types/lucene-query-string-builder/index.d.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * Lucene Query String Builder
+ */
+export as namespace lucene;
```

Comment:
I don't think this is required; the package isn't UMD, no? https://unpkg.com/browse/lucene-query-string-builder@1.0.8/index.js does not modify globals or anything.

Reply:
Thank you for pointing that out. I will get rid of those.